add intro and example of stain norm docs (#506)
histolab/histolab: docs/api/stain_normalizer.rst

Stain Normalizer
================

Digital pathology images can present strong color differences due to diverse acquisition techniques (e.g., scanners, laboratory equipment and procedures)

> The ``stain_normalizer`` module collects methods for to transfer the staining style of an image (target) to another image (source).

Reviewer: ```suggestion
The ``stain_normalizer`` module collects methods to transfer the staining style of an image (target) to another image (source).
```

    :inherited-members:

.. toctree::

Usage
----------

> The stain normalization methods can be used to tranfer the staining's appereance of any histology image. First, we need to establish the image to be used as style reference, and the image to be transformed accordigly:

Reviewer: ```suggestion
The stain normalization methods can be used to tranfser the staining's appearance of any histology image. First, we need to establish the image to be used as a style reference, and the image to be transformed accordingly:
```

> The stain normalization methods can be used to tranfser the staining's appearance of any histology image. First, we need to establish the image to be used as a style reference, and the image to be transformed accordingly:

Reviewer: ```suggestion
The stain normalization methods can be used to transfer the staining's appearance of any histology image. First, we need to establish the image to be used as a style reference, and the image to be transformed accordingly:
```

.. code-block:: ipython3

   from histolab.stain_normalizer import ReinhardStainNormalizer
   from PIL import Image
   target_image = Image.open("/path/target/img/img2.png")
   target_image

.. figure:: https://user-images.githubusercontent.com/31658006/212924301-c80f454e-f99a-4479-9852-6ef988c078aa.png
   :align: center
   :width: 200
   :figclass: align-center

.. code-block:: ipython3

   source_image = Image.open("/path/img/to/normalize/img1.png")
   source_image

.. figure:: https://user-images.githubusercontent.com/31658006/212924179-a85573b6-1bb3-4f9b-a8ab-00a26b1d652e.png
   :align: center
   :width: 200
   :figclass: align-center

> The chosen stain normalization method must be then fit on the target image and then applied to the source image:

Reviewer: ```suggestion
The chosen stain normalization method must be first fit on the target image and then applied to the source image:
```
